characteristics.pl

```diff
@@ -0,0 +1,23 @@
+:- ['utils'].
+:- dynamic achivement/1, visited/1.
+
+characteristic(explorer) :-
+	characteristic(curious),
+	characteristic(strong).
+
+characteristic(curious) :-
+	has(hero, location_map).
+	
+characteristic(strong) :-
+	has(hero, sword).
+	
+getCharacteristics :-
```

Comment:
You are right, I sometimes use different naming convention after changing the programming language, I will fix it :)